src/Database/Relations/DeferOneOrMany.php

```diff
@@ -37,6 +37,9 @@ public function withDeferred($sessionKey)
 
             // Bind (Add)
             $query = $query->orWhereExists(function($query) use ($sessionKey) {
+                if(DbDongle::getDriver() == 'sqlsrv') {
```

Comment:
Without this, the query will not execute on sql server. it needs the inital select *